README.md

```diff
@@ -123,21 +123,14 @@ tox -e static
 tox -e cov
 ```
 
-### build and upload a new release
+### Releasing
+To release a new version of this library, authorized developers should tag the
+desired commit using [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
+prepended with "v" and push the tag. A Github workflow will then automatically
+release the version to PyPI.
 
-- update all occurrences of `__version__`
```

Comment:
yes, except you do also still have to update two ocurrences of `version` - one in setup.py and the other in __init__.py
for example yesterday I forgot to do this and the github release action failed because it was trying to push to old release.

Reply:
it would be good, if possible, to reduce this to one occurrence of `version`, whether that is __init__.py or setup.py